Databricks/Dockerfile

```diff
@@ -59,6 +60,7 @@ RUN set -ex && \
         && /databricks/python3/bin/pip install \
             databricks-cli \
             ipython \
+        && if [ "$DATABRICKS_RUNTIME_VERSION" = "11.3" ]; then set -ex && /databricks/python3/bin/pip install ipykernel==6.12.1 matplotlib==3.4.3; fi \
```

Comment:
so this requires us to deploy 2 different docker images right?  I'm wondering if we can do this dynamically in init script or something. We would need to look if we can get runtime fromthere

Reply:
Yes, by this change, it'll ask us to deploy 2 different docker images.
Seems we only define one environment variable "DATABRICKS_RUNTIME_VERSION" to be 10.4 or 11.3.
I may verify if it can work by defining in the init script or the environment variables in advanced options.
If it can, we may be able to deploy only one. But the 10.4 image will install ipykernel with the new version and matplotlib, not sure if it'll be a problem on 10.4.

Reply:
By removing the DATABRICKS_RUNTIME_VERSION from the Dockerfile, I verified the DB runtime version was actually decided by the version selected when creating the DB cluster.
So, no need to define this variable.
And it also makes we only deploy one image for both.

Reply:
you are just saying that ipykernel==6.12.1  works for both 10.4 and 11.3 then?  what version is 10.4 actually using?